src/ui-utils/clients.ts

```diff
@@ -23,27 +29,35 @@ export function isLaunchpadClient(
   return client instanceof LaunchpadClient;
 }
 
+export function isStargateSigningClient(
+  signingClient: LaunchpadSigningClient | StargateSigningClient | null,
+): signingClient is StargateSigningClient {
+  return signingClient instanceof StargateSigningClient;
+}
+
+export function isLaunchpadSigningClient(
+  signingClient: LaunchpadClient | StargateClient | null,
+): signingClient is LaunchpadSigningClient {
+  return signingClient instanceof LaunchpadSigningClient;
+}
+
 export function generateMnemonic(): string {
   return Bip39.encode(Random.getBytes(16)).toString();
 }
 
-export function loadOrCreateMnemonic(): string {
+export function loadOrCreateMnemonic(mnemonic?: string): string {
```

Comment:
This should be called something like `importOrGenerateMnemonic` as it does not load anymore